vpdq/vpdq-realese.py

```diff
@@ -0,0 +1,63 @@
+from distutils.command.sdist import sdist
+from platform import release
+import shutil
+from pathlib import Path
+import os
+import subprocess
+import argparse
+
+
+DIR = Path(__file__).parent
+PARENTDIR = Path(__file__).parents[1]
+
+SETUP = "setup.py"
+MANIFEST = "MANIFEST.in"
+DIST = "dist"
+
+
+def get_argparse() -> argparse.ArgumentParser:
+    ap = argparse.ArgumentParser(description=__doc__)
```

Comment:
blocking: This doesn't do anything, because your file has no docstring. Either add a docstring or remove this